plot.go

```diff
@@ -106,8 +106,10 @@ func New() (*Plot, error) {
 		Legend:          legend,
 	}
 	p.Title.TextStyle = draw.TextStyle{
-		Color: color.Black,
-		Font:  titleFont,
+		Color:  color.Black,
+		Font:   titleFont,
+		XAlign: -0.5,
```

Comment:
It also occurs to me that if we make such a function, we can pass _that_ as the argument to makeAxis:

`func makeAxis(tickAlign func(*TextStyle)) {…}`